static/js/App.tsx

```diff
@@ -26,34 +29,76 @@ const App: FC = () => {
     <Router>
       <QueryClientProvider client={queryClient}>
         <SharedNotifyProvider>
-          <div className="l-application" role="presentation">
-            <Navigation />
-            <Routes>
-              <Route path="/" element={<InstanceList />} />
-              <Route path="/ui" element={<InstanceList />} />
-              <Route path="/ui/instances" element={<InstanceList />} />
-              <Route path="/ui/instances/:name" element={<InstanceDetail />} />
-              <Route
-                path="/ui/instances/:name/:activeTab"
-                element={<InstanceDetail />}
-              />
-              <Route path="/ui/images" element={<ImageList />} />
-              <Route path="/ui/profiles" element={<ProfileList />} />
-              <Route path="/ui/profiles/:name" element={<ProfileDetail />} />
-              <Route path="/ui/networks" element={<NetworkList />} />
-              <Route path="/ui/projects" element={<ProjectList />} />
-              <Route path="/ui/storages" element={<StorageList />} />
-              <Route path="/ui/cluster" element={<ClusterList />} />
-              <Route path="/ui/warnings" element={<WarningList />} />
-              <Route path="/ui/settings" element={<Settings />} />
-              <Route
-                path="/ui/certificates"
-                element={<CertificateGenerate />}
-              />
-              <Route path="*" element={<NoMatch />} />
-            </Routes>
-            <Panels />
-          </div>
+          <AuthProvider>
```

Comment:
Not a big deal, but you could create an `index.tsx` and move all providers there, which would keep this file cleaner. And this is a more usual way to handle the file structure.
https://github.com/canonical/anbox-cloud-dashboard/blob/main/static/js/index.tsx

Reply:
Done